scripts/run.yaml

```
@@ -14,3 +14,8 @@ runs:
     command: /usr/bin/sqlite3 
     memory: 64
     networking: False
+  - name: http_server
```

```suggestion
  - name: http_server-cpp
```

To differentiate it from other servers.